www/friendly.html.pm

```diff
@@ -0,0 +1,26 @@
+#lang pollen
+
+◊top-section{
+◊span[#:id "logo" #:style "font-size:2.3rem;white-space:nowrap;"]{◊link["index.html"]{◊img[#:src "img/racket-logo.svg" #:class "logo"] Racket} ◊span[#:id "tagline" #:class "disappearing" #:style "font-size:70%;color:gray;white-space:nowrap;margin-left:0.2rem;"]{}}
+
+◊div{
+◊link[#:class "top-button disappearing-late" #:id "docs" "https://docs.racket-lang.org/"]{docs}
+
+◊link[#:class "top-button disappearing-late" #:id "packages" "https://pkgs.racket-lang.org/"]{packages}
+
+◊link[#:class "top-button disappearing-late" #:id "download" "https://download.racket-lang.org/"]{download}
+}
+}
+
+
+◊section[#:class "one-column-body-text"]{
+Friendly Environment Policy
+
+The Racket community aims to improve the world through programming. It started with the goal of introducing everyone to the wonderful world of program design, with a spirit of full inclusion and no emphasis on any specific group.  Over time it has grown into a full-fledged professional community with a well-known reputation for helpfulness and openness on its on-line communication channels. Racket project management wants to encourage and preserve that open exchange of ideas.
+
+This spirit requires an environment that enables all to participate without fear of personal harassment. We define harassment as unwelcome or hostile behavior, that is, behavior that focuses on people instead of ideas. ◊link["http://www.acm.org/sigs/volunteer_resources/officers_manual/anti-harassment-policy"]{The ACM's anti-harassment policy} lists a variety of specific unacceptable factors and behaviors. Racket project management considers responses such as “just joking,” or “teasing,” or being “playful” as unacceptable.
```

Comment:
This link does not go anywhere useful for me (I just get redirected to https://www.acm.org/special-interest-groups/volunteer-resources, which does not have any obvious link to the policy that I could find), which seems like a pretty big problem.

Reply:
I think it's now at https://www.acm.org/about-acm/policy-against-harassment

Reply:
There's an "ACM Policies" box on the lower right. But that took me a while to find, too, so probably a more direct link is better.

Reply:
Thanks! Since the link seems questionably stable, I think it would be better if the salient points were quoted or paraphrased on this page directly, but maybe it isn’t a big problem.

Reply:
> We define harassment as unwelcome or hostile behavior, that is, behavior that focuses on people instead of ideas.

I would reword this sentence. While I think it means well, as-worded it essentially says “do not criticize someone for their personal beliefs.” Though quite reasonable in a vacuum, this and similar sentiments (such as “be nice” or “be civil”) have sadly been used to take action against people who call attention to discrimination, especially if that discrimination took place in a different forum or community.

I don’t think anyone has any reason to worry about that problem in the Racket community, but those who benefit most from a Code of Conduct don’t have the luxury of already knowing that context. Most people will probably interpret it charitably, but it helps to leave no room for doubt.

I like the way [Rust’s Code of Conduct](https://www.rust-lang.org/policies/code-of-conduct) phrases this:

> We will exclude you from interaction if you insult, demean or harass anyone. […] In particular, we don’t tolerate behavior that excludes people in socially marginalized groups. Private harassment is also unacceptable.

It’s short and to the point, but it’s also very clear.

Reply:
I don't like the Rust sentence at all. It is overly aggressive and focused on specific groups. 

We care that every *individual* is treated properly, period. Much stronger, while not being aggressive againt, or focused on, any group.

Reply:
There is no way to properly address harassment without addressing the fact that some groups of people are far more likely to be harassed than others.

Reply:
There is. Respect every individual. Every single one of them. The End.

Reply:
Matthias: I am not suggesting copying the Rust Code of Conduct wholesale. I do suggest incorporating the three sentences I quoted. Rewording them slightly is fine, but muddying their meaning is not.

The first sentence reads, “We will exclude you from interaction if you insult, demean or harass **anyone**.” It is very clear. You may not find the second sentence necessary, but it is hardly offensive, and it will give some people who particularly need it a feeling of additional safety.

With all due respect: don’t die on this particular hill. It doesn’t really affect you. I guarantee it matters quite a lot more to some people than you find it philosophically unnecessary.

Reply:
It affects me in very serious ways, and that's precisely why I don't want to speak of goops, ever. As a non-liberal I am routinely attacked---including at a RacketCon a couple of years back. I don't seek help, I don't advertise this, I cope but I ascribe all of this to the group think that is so pervasive in the modern and especially the tech world. 

You have already exclude me from 'users'. If you wish to exclude me from other venues too, this is the way to go.

Reply:
> As a non-liberal I am routinely attacked---including at a RacketCon a couple of years back. I don't seek help, I don't advertise this, I cope but I ascribe all of this to the group think that is so pervasive in the modern and especially the tech world.

If you feel you have been harassed, it seems like this very document provides some recommendations on how that should be handled, with or without my suggestions.

But that is really immaterial. Clearly, you agree that “socially marginalized groups” should be able to participate in Racket community spaces without fear of harassment, seeing as they are a subset of “everyone.” Offering marginalized people—that is, people who are discriminated against more often *by the definition of the word “marginalized”*—extra reassurance that they are, in fact, welcome should not be a political issue. If you feel that sentence is an “attack” on “non-liberals,” I am not sure what to say.

Reply:
I heartily agree with Matthias. We want to foster respect and kindness and love and charity for all people regardless of their membership in any particular groups. No groups deserve more or less respect or protection. By including this sentence, we are favoring some over others.

Reply:
I agree with @lexi-lambda. I agree we should focus on respect for individual people... that's what the first sentence is for. It applies to people whether they're part of an often-marginalized group or not. But that doesn't make the next part unnecessary. I think we should give extra assurance that we don't allow harassment against people in marginalized groups in a follow-up sentence.